recipes/pomoxis/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "pomoxis" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4001024e7c2b200e4924c2924f3c423e3f7464b51501619b3c310f51b5120df9
+
+build:
+  number: 1
+  skip: True # [not py35 or not py36]
```

Comment:
You can remove this as you pinned the version in the host and run section.
Instead you could try to add `noarch: python`